lib/matplotlib/tests/test_legend.py

```diff
@@ -1438,15 +1439,90 @@ def test_legend_text():
 
 
 def test_boxplot_labels():
```

Comment:
This test includes various calls that do not affect the outcome of the test e.g. setting colours/titles and adding the legends.  As they don't affect the outcome, I think it would be better to strip them out so the code is simpler and more efficient.